fractal-puppy/Engine
# Feature/music buttons script

Menu Volume buttons binded to the main theme music volume.

TO TEST:
open menuEscene  > options > play with the music volume buttons and check that the main theme volume lowers or rises
